keyboards/novelkeys/nk87/config_led.c

```diff
@@ -0,0 +1,170 @@
+/* Copyright 2022 Kirill Malgichev (@makirill)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "config_led.h"
+#include "quantum.h"
+
+#include "indicator.h"
+
+
+#ifdef RGB_MATRIX_ENABLE
```

Comment:
This should all be moved to `nk87.c`, and this file should be removed.

Reply:
Fixed with db9256d949d2c36a8ee1e34dff45a2547ceb6f5e